src/hooks/use-dataset-pieces.ts

```diff
@@ -110,13 +110,14 @@ export const useDatasetPieces = () => {
               fileSize,
               cid: ipfsRootCid,
               pieceCid,
-              providerName: providerInfo.name || (providerInfo.id ? String(providerInfo.id) : undefined),
+              providerName: providerInfo.name || 'unknown',
               datasetId: String(storageContext.dataSetId),
-              providerId: providerInfo.id ? String(providerInfo.id) : undefined,
-              serviceURL: providerInfo.products?.PDP?.data?.serviceURL,
-              network: wallet?.status === 'ready' ? wallet.data.network : undefined,
+              providerId: providerInfo.id.toString(),
```

Comment:
I'm just checking to make sure providerInfo.id is definitely not going to be non-null so that this doesn't throw an error.